cli/src/cmd/forge/script/cmd.rs

```diff
@@ -61,56 +63,77 @@ impl ScriptArgs {
+                    Err(_) if broadcasted => ScriptSequence::load(
+                        &script_config.config,
+                        &self.sig,
+                        &target,
+                        chain,
+                        false,
+                    )?,
+                    Err(err) => eyre::bail!(err),
+                };
+
+                receipts::wait_for_pending(provider, &mut deployment_sequence).await?;
+
+                if self.resume {
+                    self.send_transactions(
+                        &mut deployment_sequence,
+                        &fork_url,
+                        &result.script_wallets,
+                    )
                     .await?;
-            }
+                }
 
-            if self.verify {
-                // We might have predeployed libraries from the broadcasting, so we need to relink
-                // the contracts with them, since their mapping is not included in the solc cache
-                // files.
-                let BuildOutput { highlevel_known_contracts, .. } = self.link(
-                    project,
-                    default_known_contracts,
-                    Libraries::parse(&deployment_sequence.libraries)?,
-                    script_config.config.sender, // irrelevant, since we're not creating any
-                    U256::zero(),                // irrelevant, since we're not creating any
-                )?;
+                if self.verify {
+                    // We might have predeployed libraries from the broadcasting, so we need to
+                    // relink the contracts with them, since their mapping is
+                    // not included in the solc cache files.
+                    let BuildOutput { highlevel_known_contracts, .. } = self.link(
+                        project,
+                        default_known_contracts,
+                        Libraries::parse(&deployment_sequence.libraries)?,
+                        script_config.config.sender, // irrelevant, since we're not creating any
+                        U256::zero(),                // irrelevant, since we're not creating any
+                    )?;
 
-                verify.known_contracts = flatten_contracts(&highlevel_known_contracts, false);
+                    verify.known_contracts = flatten_contracts(&highlevel_known_contracts, false);
 
-                deployment_sequence.verify_contracts(verify, chain).await?;
+                    deployment_sequence.verify_contracts(&script_config.config, verify).await?;
+                }
+            } else {
+                let multi =
+                    MultiChainSequence::load(&script_config.config.broadcast, &self.sig, &target)?;
+
+                self.multi_chain_deployment(
+                    multi,
+                    libraries,
+                    &script_config.config,
+                    result.script_wallets,
+                    verify,
+                )
+                .await?;
```

Comment:
`if self.debug { } else {}` could be reduced to 

`if self.debug {...; return}`? to get rid of the else branch indent?